### PR TITLE
made_some_crayons_brighter

### PR DESCRIPTION
**Pull request by TruePaIntGit**
## About the PR

I made a green, blue, purpule crayons brighter

## Why / Balance

Resolves https://github.com/space-wizards/space-station-14/issues/31615

## Technical details

No C#

## Media

noname

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Color of some crayons.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Fun/crayons.yml [hunk 1]
```diff
@@ -290,4 +290,4 @@
         whitelist:
           tags:
           - CrayonWhite
-  - type: Appearance
+  - type: Appearance
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Fun/crayons.yml:293]:
```suggestion
  - type: Appearance

```

Comment (slarticodefast) [reply Resources/Prototypes/Entities/Objects/Fun/crayons.yml:293]: How does this even keep happening? Is it a wrong .editorconfig setting?

Comment (lzk228) [reply Resources/Prototypes/Entities/Objects/Fun/crayons.yml:293]: i suppose so

**Comment by slarticodefast** [comment]
Small tip: If you write "Fixes #31615" or "Resolves #31615" in your PR description GitHub will automatically link and close the mentioned issue when this is merged.

**Comment by Graded5** [comment]
YOOOO they made blue blue!

**Comment by slarticodefast** [comment]
Can you edit the respective crayon icons and the layers for the crayon box so they match up correctly?
grafik

**Comment by slarticodefast** [comment]
Are you coming back to this?

**Comment by slarticodefast** [comment]
Leave a comment in case you want to continue working on this and I will re-open the PR.

**Pull request closed without merging**